.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,145 @@
+id: 
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner: 
+resource: repository
+disabled: false
+where: 
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+    - description: 

```

Comment:
The description didn't get filled in. Should it have been filled from the `taskName`, @JohannesLampel?

https://github.com/dotnet/docs/blob/0135bf9df107eb22edcf0f0096aea87a0399712c/.github/fabricbot.json#L522-L530

Reply:
I can look into this and try to add it. If it is not a big change, we can provide this.

Reply:
I pushed a change to resolve this directly in this PR. It looks like it was just scheduled searches that weren't getting the descriptions filled in, @JohannesLampel.